src/components/post/Post.js

```diff
@@ -0,0 +1,20 @@
+import styles from './Post.module.css'
+import React from 'react';
+import {Link} from "react-router-dom";
+import formatNumbers from "../../helpers/formatNumbers";
+import limitStringLength from "../../helpers/limitStringLength";
+
+function Post({title, titleUrl, subreddit, subredditPrefix, comments, ups}) {
+    return (
+        <article className={styles.article}>
+            <a className={styles["post-title"]} href={titleUrl}>{limitStringLength(title)}</a>
+            <span>
+                <Link className={styles.link}
+                      to={`/subreddit/${subreddit}`}>{subredditPrefix}</Link>
+                <p>Comments {formatNumbers(comments)} - UPS {formatNumbers(ups)}</p>
+            </span>
```

Comment:
Dit span element heb je zover ik kan zien niet nodig:
```suggestion
            <Link className={styles.link}
                  to={`/subreddit/${subreddit}`}>{subredditPrefix}</Link>
            <p>Comments {formatNumbers(comments)} - UPS {formatNumbers(ups)}</p>
```